src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -74,16 +95,22 @@ public LocalDate addWeeks(LocalDate localDate, Integer numberOfWeeks) {
      * - "someDate is today" - если someDate - сегодня
      */
     public String beforeOrAfter(LocalDate someDate) {
-        return someDate + "is today";
+        return localDate.equals(someDate) ? someDate.toString()
```

Comment:
you don't need to call toString() here, since you're using concat with +, it will do that automatically